brave-jersey2/pom.xml

```diff
@@ -34,7 +34,7 @@
 
         <dependency>
             <groupId>org.glassfish.jersey.test-framework.providers</groupId>
-            <artifactId>jersey-test-framework-provider-grizzly2</artifactId>
+            <artifactId>jersey-test-framework-provider-inmemory</artifactId>
```

Comment:
reverts this back as we don't need grizzly anymore (as we're using a jetty container now)